main/reference/vstorage-ref.md

```diff
@@ -1,13 +1,128 @@
-# VStorage Reference
+# VStorage
```

Comment:
As the "See also" list at the bottom of the page shows, we have more than one page on VStorage. The heading should distinguish this one somehow. If it's not reference docs for vstorage, what is it?

Do we move reference docs for vstorage elsewhere? Or do we just not have it any more?